snuba/query/parser/expressions.py

```diff
@@ -52,6 +56,44 @@ def visit_function_name(self, node: Node, visited_children: Iterable[Any]) -> st
     def visit_column_name(self, node: Node, visited_children: Iterable[Any]) -> Column:
         return Column(None, None, node.text)
 
+    def visit_empty(
+        self, node: Node, visited_children: Iterable[Any]
+    ) -> Optional[Expression]:
+        return None
+
+    def visit_top_arithm_expression(
+        self, node: Node, visited_children: Tuple[Any, Expression, Any]
+    ) -> Expression:
+        _, exp, _ = visited_children
+
+        return exp
+
+    def visit_low_pri_arithmetic(
+        self,
+        node: Node,
+        visited_children: Tuple[Any, Expression, Any, List[Expression]],
+    ) -> Expression:
+        _, term, _, exp = visited_children
+
+        if exp is None:
+            return term
+        else:
+            return plus(term, exp[1])
```

Comment:
Do you expect to use `exp[1]` becasue `exp[0]` is the "+" string ?
If yes, mind adding a quick comment above to explain this? Also the type inn the signature of the method (`List[Expression]`) is wrong. It is more likely going to be `Tuple[str, Expression]`

Reply:
Added comment and modified type hint.

Reply:
I think the point of this comment is to explain why it is not exp[0] that you are returning. So explain that exp[0] is the string that represent the operator.